azerothcore/azerothcore-wotlk
# fix(scripts/SlavePens): Rework Naturalist Bite event

Co-Authored by: @Nyeriah

## Changes Proposed:
-  Corrects summon group, add their line
-  Ports from cpp to DB
- Adds Bite's waypoints

## Issues Addressed:

- Makes Progress towards https://github.com/azerothcore/azerothcore-wotlk/issues/8559
- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/14259
- Closes https://github.com/chromiecraft/chromiecraft/issues/4573

## SOURCE:


## Tests Performed:

- 
- 


## How to Test the Changes:


1. `.go c id 17893`
2. Start Event
3. On end, see if gossip text changes to "Now that the rude interruption is over..." and Option to "Naturalist, please grant me your boon."

## Known Issues and TODO List:


- [ ] Cannot get Conditions to work
- [ ] Even with IMMUNE_TO_NPC removed, Naturalist Bite won't engage in combat with summon group


## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
